.github/PULL_REQUEST_TEMPLATE/pull_request_template_api.md

```diff
@@ -3,9 +3,9 @@
 ### Pre-requisites
 
 - [ ] I have gone through the Contributing guidelines for [Submitting a Pull Request (PR)](../../CONTRIBUTING.md#submitting-a-pull-request-pr) and ensured that this is not a duplicate PR.
-- [ ] I have performed preliminary testing using the [test suite](../../apps/api/osmo-notify.postman_collection.json) to ensure that any existing features are not impacted and any new features are working as expected.
+- [ ] I have performed preliminary testing using the [test suite](../../apps/api/osmoX.postman_collection.json) to ensure that any existing features are not impacted and any new features are working as expected.
```

Comment:
You can fix the casing for this as `OsmoX.postman_collection.json`

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b